### PR TITLE
crud  operations

### PR DESCRIPTION
**Pull request by samir5687**

### CODE REVIEW

File: crud_file.c [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+typedef struct {
+    int id;
+    char name[50];
+    int age;
+} User;
+
+
+void createUser() {
+    FILE *file = fopen(FILE_NAME, "a");
```
Comment (intimetec-shiv-maheshwari) [inline crud_file.c:15]: Please handle the all the corner case scenarios in addition of a new user.
Eg. I am able to create 2 users with same id

There are many more scenarios where your code is failing. Test it thoroughly.

File: crud_file.c [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    if (found) printf("User deleted successfully.\n");
+    else printf("User not found.\n");
+}
+
+int main() {
+    int choice;
+    while (1) {
+        printf("\n1. Create User\n2. Read Users\n3. Update User\n4. Delete User\n5. Exit\n");
+        printf("Enter choice: ");
+        scanf("%d", &choice);
```
Comment (intimetec-shiv-maheshwari) [inline crud_file.c:108]: Handle the corner case in handling the inputs of this choice variable.
Code is failing at multiple instances.
